mysql-test/main/mysqltest.test

```diff
@@ -2369,15 +2369,136 @@ file_exists $MYSQLTEST_VARDIR/tmp/test_file1.tmp;
 # ----------------------------------------------------------------------------
 # test for copy_file
 # ----------------------------------------------------------------------------
+
+# ----------------------------------------------------------------------------
+# Test #1: Basic file copying
+# ----------------------------------------------------------------------------
 --write_file $MYSQLTEST_VARDIR/tmp/file1.tmp
 file1
 EOF
 
 copy_file $MYSQLTEST_VARDIR/tmp/file1.tmp $MYSQLTEST_VARDIR/tmp/file2.tmp;
 file_exists $MYSQLTEST_VARDIR/tmp/file2.tmp;
+
+# ----------------------------------------------------------------------------
+# Test #2: Copy with wildcard
+# ----------------------------------------------------------------------------
+mkdir $MYSQLTEST_VARDIR/tmp/testdir;
+copy_file $MYSQLTEST_VARDIR/tmp/file*.tmp $MYSQLTEST_VARDIR/tmp/testdir;
```

Comment:
is there a test with the source is not a wildcard and the destination is a directory?